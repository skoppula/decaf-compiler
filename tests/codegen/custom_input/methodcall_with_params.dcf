callout printf;

void foo (int paramOne,int paramTwo) {
  int a;
  a = paramOne + paramTwo;

  printf ( "Should equal 5, is %ld\n", a );
}

void main ( ) {
  int two;
  int three;
  two = 2;
  three = 3;

  foo(two , three);
}

