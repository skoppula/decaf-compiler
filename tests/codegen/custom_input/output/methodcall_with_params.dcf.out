Should equal 5, is 5
